system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs

```diff
@@ -817,49 +817,14 @@ pub type UncheckedExtrinsic =
 	generic::UncheckedExtrinsic<Address, RuntimeCall, Signature, SignedExtra>;
 /// Migrations to apply on runtime upgrade.
 pub type Migrations =
-	(pallet_collator_selection::migration::v1::MigrateToV1<Runtime>, InitStorageVersions);
+	frame_support::migrations::VersionedMigration<0, 1, UniquesMigration, Uniques, RocksDbWeight>;
```

Comment:
If the `InitStorageVersions` was executed, we need to reset the `Uniques` on-chain version back to 0 before running this.

Reply:
No this init was never executed on chain. This was the reason the CI was failing for this pr.

Reply:
I wonder why it was not, I can see it here - https://github.com/polkadot-fellows/runtimes/blob/v1.0.0/system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs#L799

Reply:
Yes the migration was there. But the init of Uniques was added here: https://github.com/polkadot-fellows/runtimes/pull/87/files#diff-2dc481414ffabfb3b0b2c24f3c0cfc8a8b3ed6b56689a696d02be635795f5cb6R857